llvm-external-projects/iree-dialects/lib/Dialect/LinalgExt/Passes/ConvertConv2DToWinograd.cpp

```diff
@@ -68,6 +68,8 @@ static DenseElementsAttr foldFilterTransform(
   const int &ic = shape[2];
   const int &oc = shape[3];
   const int64_t numElements = inputTileSize * inputTileSize * ic * oc;
+  FloatType floatType = elementType.cast<FloatType>();
+  bool losesInfo;
```

Comment:
move the declaration to where it's used, i.e., line 92?

Reply:
sure will move